### DISCUSSION TITLE
How to add proxy configuration in Qfield for Windows

### DISCUSSION THREAD

**Discussion by llgis** [opengisch/QField#5868] [Q&A]
Hello, we would like to use **QField for Windows**. 

Unfortunately, we are currently unable to access WMS services from either the internet or the intranet. 
We have a network-wide proxy in place that controls the traffic. In order to **access the proxy**, authentication needs to be configured in QField. 

We added

[proxy]
proxyExcludedUrls=http://URL_TO_WMS_HOST
authcfg=
proxyEnabled=true
proxyHost=URL_TO_PROXY
proxyPort=8080
proxyUser=USER
proxyPassword=PASSWD
proxyType=2

in "C:\Program Files\QField\usr\share\qgis\resources\qgis_global_settings.ini"

but it did not work.

**Where can I set the setting for the proxy in qfield?** 

Thank you for your support!!!

**Comment by nirvn** [comment]
@llgis , please add this to the ideas platform, that'll get a chance to be upvoted and seen by others who might be willing to sponsor this enhancement: https://ideas.qfield.org/

**Reply by llgis** [reply]
i added the idea. it would be great to set the proxy settings in qfield similar to qgis.
![grafik](https://github.com/user-attachments/assets/48540f46-aadb-4e42-ab36-0e6143ea8e48)
